build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -96,18 +97,29 @@ abstract class BuildCommitDistribution @Inject internal constructor(
             "-PtoolingApiShadedJarInstallPath=" + commitDistributionToolingApiJar.get().asFile.absolutePath,
```

Comment:
I think we should tag these builds. We can do this by passing a system property.